Dockerfile

```diff
@@ -11,7 +11,7 @@
 # TODO:
 #	- delete archives to keep small the container small
 #	- setup caching (for apt, pip, ndk, sdk and p4a recipes downloads)
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
I think we should be up to date, mainstream, but not too cutting edge, hence the choice for the last Ubuntu LTS.